reader/signals.py

```diff
@@ -105,7 +155,8 @@ def save_volume(sender, instance, **kwargs):
             quality=60,
             method=6,
         )
-        image.save(os.path.join(settings.MEDIA_ROOT, save_dir, f"{filename}.jp2"))
+        # This line crash on my server and this file does not seem be used at all.
+        # image.save(os.path.join(settings.MEDIA_ROOT, save_dir, f"{filename}.jp2"))
```

Comment:
Yea JPEG2000 was supposed to be an alternative to devices that didn't support `.webp` but we never ended up using it. Seems fine to remove.